Dockerfile.prebuilt

```diff
@@ -0,0 +1,84 @@
+# Dockerfile.prebuilt
+#
+# This Dockerfile is designed for pre-built binaries (built outside Docker).
+# It bypasses the Zig HTTP connection pool bug by copying pre-built artifacts.
+#
+# Usage:
+#   1. Build natively: zig build -Doptimize=ReleaseFast -Dgit_version="$(git rev-parse --short HEAD)"
+#   2. Build image: docker build -f Dockerfile.prebuilt -t zeam:latest .
+#
+# This approach is used for multi-arch builds on CI/VMs to avoid intermittent
+# Zig HTTP failures during dependency fetching.
+
+# Runtime prep stage - get the necessary libraries
+FROM ubuntu:24.04 AS runtime-prep
+ARG TARGETARCH
+
+# Install SSL certificates
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy pre-built binaries and resources
+COPY zig-out/ /app/zig-out/
+COPY resources/ /app/resources/
+
+# Create a script to copy the right libraries based on architecture
+RUN mkdir -p /runtime-libs && \
+    case "$TARGETARCH" in \
+        amd64) \
+            LIBDIR="x86_64-linux-gnu" && \
+            LDSO="/lib64/ld-linux-x86-64.so.2" \
+            ;; \
+        arm64) \
+            LIBDIR="aarch64-linux-gnu" && \
+            LDSO="/lib/ld-linux-aarch64.so.1" \
+            ;; \
+        arm) \
+            LIBDIR="arm-linux-gnueabihf" && \
+            LDSO="/lib/ld-linux-armhf.so.3" \
+            ;; \
+        386) \
+            LIBDIR="i386-linux-gnu" && \
+            LDSO="/lib/ld-linux.so.2" \
+            ;; \
+        riscv64) \
+            LIBDIR="riscv64-linux-gnu" && \
+            LDSO="/lib/ld-linux-riscv64-lp64d.so.1" \
+            ;; \
+        *) \
+            echo "Unsupported architecture: $TARGETARCH" && exit 1 \
+            ;; \
+    esac && \
+    if [ -d "/lib/$LIBDIR" ]; then \
+        mkdir -p "/runtime-libs/lib/$LIBDIR" && \
+        for lib in libc.so.6 libm.so.6 libpthread.so.0 libdl.so.2 librt.so.1 libgcc_s.so.1 libstdc++.so.6; do \
+            [ -f "/lib/$LIBDIR/$lib" ] && cp -L "/lib/$LIBDIR/$lib" "/runtime-libs/lib/$LIBDIR/" || true; \
+        done; \
+    fi && \
+    if [ -f "$LDSO" ]; then \
+        mkdir -p "/runtime-libs$(dirname $LDSO)" && \
+        cp -L "$LDSO" "/runtime-libs$LDSO"; \
+    fi
+
+# Runtime stage - using scratch for minimal size
+FROM scratch AS runtime
+
+# Copy the architecture-specific libraries and loader
+COPY --from=runtime-prep /runtime-libs/ /
+
+# Copy SSL certificates for HTTPS
+COPY --from=runtime-prep /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```

Comment:
Can't you just apt install these ? The http issues are only for zig, or also for the system packaging as well?